falcosidekick/values.yaml

```diff
@@ -96,6 +96,10 @@ config:
     accesskeyid: ""
     secretaccesskey: ""
     region : ""
+    cloudwatchlogs:
+      loggroup : ""
+      logstream : ""
+      minimumpriority: ""
```

Comment:
I put this block in front of lambda because I wanted to sort it alphabetically as much as possible.

Reply:
No issue with that, I like when things are ordered